### ISSUE TITLE
"attempt to toggle disabled state of second handle using ya-no-ui-slider-handle2-disabled attribute in one-handle slider"

### ISSUE THREAD

**Issue by rettgerst** [Yankovsky/nouislider-angular#22]
nouislider-angular is giving this error:

> Warning: attempt to toggle disabled state of second handle using ya-no-ui-slider-handle2-disabled attribute in one-handle slider, nouislider-angular is ignoring such call.

but I'm not using that attribute. my containers look like this:

``` html
<div ya-no-ui-slider="sliderOptions['nodeSep']" ya-no-ui-slider-events="sliderHandlers['nodeSep']"></div>
```

and they're getting passed the options:

``` js
var commonSliderOptions = {
    start: 10,
    range: {min: 0, max: 100},
    connect: 'lower'
};
```

**Comment by Yankovsky** [comment]
@rettgerst please check if you are using latest version, I fixed this bug in 2.6.1

**Comment by Yankovsky** [comment]
Fixed in newer versions
